### ISSUE TITLE
PNASNet: Inconsistence between code and paper.

### ISSUE THREAD

**Issue by gyang274** [tensorflow/models#4729]
Follow the [PNASNet paper](https://arxiv.org/pdf/1712.00559.pdf), Figure 1: 
![PNASNet-5 Cell Structure](https://user-images.githubusercontent.com/13155537/42476460-07bf9526-839c-11e8-914d-5ce7c7df7a8c.png)

The output from each normal cell would be, say, 4 * num_conv_filters, e.g., 4 * 216 = 864.

However, in the official model released

https://github.com/tensorflow/models/blob/3a05570f8d5845a4d56a078db8c32fc82465197f/research/slim/nets/nasnet/pnasnet.py#L192

The output from each normal cell is, say, 5 * num_conv_filters, e.g., 5 * 216 = 1080.

The difference is that in the released code, the output of comb_iter2 (sep5x5 + sep3x3) is concat into output, whereas not in paper.

In summary, Figure 1 in the paper shall be revised to
![PNASNet-5 Cell Structure - Revised](https://user-images.githubusercontent.com/13155537/42477181-323bccb4-839e-11e8-82b9-78c52607a86c.png)

Note the red line added.

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
What is the top-level directory of the model you are using
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by gyang274** [comment]
What is the top-level directory of the model you are using: tensorflow/models
Have I written custom code: No
OS Platform and Distribution: N/A
TensorFlow installed from: N/A
TensorFlow version: N/A
Bazel version: N/A
CUDA/cuDNN version: N/A
GPU model and memory: N/A
Exact command to reproduce: N/A
